src/components/BusinessDashboard/BusinessDashboard.jsx

```diff
@@ -0,0 +1,96 @@
+import './BusinessDashboard.module.css';
+import { useBackend } from '../../contexts/BackendContext';
+import { useEffect, useState } from 'react';
+import { Table, Thead, Tbody, Tr, Th, Td } from '@chakra-ui/react';
+import { Box } from '@chakra-ui/react';
+
+const BusinessDashboard = () => {
+  const { backend } = useBackend();
+  const [donationData, setDonationData] = useState([]);
+
+  // *************************************************************************
+  // CHANGE LATER: right now just making one big request for all of the
+  // rows in the fair_market_value table since it is just full of dummy data.
+  // Later, when actual data or more accurate dummy data is used, change the
+  // request to efficiently pull the prices of dry and canned dog and cat food
+  // *************************************************************************
+  const [priceData, setPriceData] = useState([]);
+
+  const [reminderData, setReminderData] = useState([]);
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        // ***********************************************************************
+        // CHANGE LATER: right now business is hardcoded to be business with id 1
+        // in the future will have to change (add parameter to component)
+        // so that we can display the dashboard for various businesses
+        // ***********************************************************************
+        const donationResponse = await backend.get('/donation/business/totals/1');
+        setDonationData(donationResponse.data[0]);
+
+        const priceResponse = await backend.get('/value');
+        setPriceData(priceResponse.data);
+
+        const reminderResponse = await backend.get('/notification/1');
+        setReminderData(reminderResponse.data);
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'backend'. Either include it or remove the dependency array.